src/models/eventmodelproxy.cpp

```diff
@@ -0,0 +1,20 @@
+/*
+    SPDX-FileCopyrightText: Lieven Hey <lieven.hey@kdab.com>
+    SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
+
+    SPDX-License-Identifier: GPL-2.0-or-later
+*/
+
+#include "eventmodelproxy.h"
+
+EventModelProxy::EventModelProxy(QObject* parent)
```

Comment:
this file shouldn't be part of the patch that touches the multiconfig widget

also, a proxy that never filters is pretty bogus conceptually

Reply:
I think I committed this accidentally